### DISCUSSION TITLE
Breadcrumbs with local/offline pages

### DISCUSSION THREAD

**Discussion by ChrisLeeThompson** [thegeeklab/hugo-geekdoc#494] [Q&A]
I am fairly new with Hugo and the Geekdoc theme. I have a question regarding the Geekdoc breadcrumbs.

I am publishing my site locally (not using localhost). The breadcrumbs work as expected, however when I click the base link (position 1), it does not load the home page. That is, it does not load index.html. Instead, it takes me to /public/. The other links work as expected.

Is there a way I can adjust breadcrumbs such that the first link takes me to index.html?

My config file is set up as:
baseURL = "/"
relativeURLs = true
uglyURLs = true

Thank you in advance for any information you may provide. And thank you for such a great Hugo theme!

**Comment by xoxys** [comment]
I'm not able to reproduce it, please create a minimal reproducible repo. Besides that, `baseURL` as the name might indicate already should be a hostname (and optional path) to the root. What's the goal of using `/` instead?

**Comment by ChrisLeeThompson** [comment]
Thank you for your response. My use of the "/" is probably showing my lack of knowledge about correct use of Hugo. When I run "hugo" to build the pages locally, I find that with relativeURLs=true, uglyURLs = true, and "/" as the baseURL, the css file loads correctly and links in the site also work correctly.

When I run the site using hugo server, then the breadcrumbs, css, links, etc, work as expected. Eventually, I would like to have my site, using the Geekdoc theme, available online and not built locally (offline). Therefore, I feel that you have answered my question.
